docs/collecting-data/collecting-from-own-applications/mobile-trackers/previous-versions/objective-c-tracker/objective-c-0-2-0/index.md

```diff
@@ -327,14 +307,11 @@ Use `trackStructuredEvent:` to track a custom event happening in your app whic
 
 Example:
 
-```
+```objc
 [t1 trackStructuredEvent:@"shop" action:@"add-to-basket" label:@"Add To Basket" property:@"pcs" value:27];
+[t1 trackStructuredEvent:@"shop" action:@"add-to-basket" label:@"Add To Basket" property:@"pcs" value:27 timestamp:1234569];
 ```
 
-\[t1 trackStructuredEvent:@"shop" action:@"add-to-basket" label:@"Add To Basket" property:@"pcs" value:27 timestamp:1234569\]
-
-;
-
 [Back to top](https://github.com/snowplow/snowplow/wiki/iOS-Tracker-v0.2#top)
```

Comment:
Can you grep for this and kill it?

Reply:
You mean kill all the `\` slashes that have been added and the weird formatting? I likely haven't caught them all so happy to find and whack them if so. If you mean something else, let me know.

Reply:
Becuase this PR wasn't crazy enough, I added: https://github.com/snowplow/snowplow.github.io/pull/73/commits/aaa48f17805571a3e568a5f0fec96533d5ccf7a4

Reply:
I meant “back to the top”, sorry!

Reply:
Oh my, the slash thing is… care to share the commands you ran? I’d rather validate those than read through it :(

Reply:
I used VS Code for this one so I could be a little careful. 

I looked for `\[`, `\]` and (I think) the regex `([0-9])\\\.` and replaced with `$1.` (So I only caught 1. 2. etc, rather than other places with decimals).